azure_pipelines/verify_msalcpp_per_pr_ios.yml

```diff
@@ -10,7 +10,7 @@ pr:
     - '*'
 
 pool:
-  vmImage: 'macOS-12'
+  vmImage: 'macOS-13'
```

Comment:
Do you mind separating the pipeline as a different PR as it is not related. Just precaution If we need to cherry pick for any issue.

Reply:
This will break the current pipeline for MSAL CPP iOS subtree update, and fix is essential. Will create a separate PR for pipeline improvements in future

Reply:
sounds good!